### PR TITLE
Upgrade GuzzleHttp to version 7

### PR DESCRIPTION
**Pull request by tomas-kulhanek**

### CODE REVIEW

**Comment by tomas-kulhanek** [comment]
On the last CS error i dont have any idea how to fix, because i have identicaly row in this file :|

**Comment by antonkomarev** [comment]
Here is a list of issues, spaces count.
https://github.styleci.io/analyses/yvxnjY

**Comment by antonkomarev** [comment]
Thank you for the contribution!

File: src/Client/YouTrackClient.php [hunk 1]
```diff
@@ -216,7 +216,7 @@ protected function buildUri(string $uri): string
     protected function buildOptions(array $params = [], array $options = []): array
     {
         $defaultOptions = [
-            'form_params' => $params,
+            'json' => $params,
```
Comment (ilzrv) [inline src/Client/YouTrackClient.php:219]: Why did you replace it?

**Pull request closed without merging**